.github/CONTRIBUTING.md

```diff
@@ -1,13 +1,98 @@
-## How to contribute
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Contributing to hipBLAS">
+  <meta name="keywords" content="ROCm, contributing, hipBLAS">
+</head>
 
-Our code contriubtion guidelines closely follows the model of [GitHub pull-requests](https://help.github.com/articles/using-pull-requests/). The hipBLAS repository follows a workflow which dictates a /master branch where releases are cut, and a
-/develop branch which serves as an integration branch for new code.
+# Contributing to hipBLAS #
+
+We welcome contributions to hipBLAS.  Please follow these details to help ensure your contributions will be successfully accepted.
+
+## Issue Discussion ##
+
+Please use the GitHub Issues tab to notify us of issues.
+
+* Use your best judgement for issue creation. If your issue is already listed, upvote the issue and
+  comment or post to provide additional details, such as how you reproduced this issue.
+* If you're not sure if your issue is the same, err on the side of caution and file your issue.
+  You can add a comment to include the issue number (and link) for the similar issue. If we evaluate
+  your issue as being the same as the existing issue, we'll close the duplicate.
+* If your issue doesn't exist, use the issue template to file a new issue.
+  * When filing an issue, be sure to provide as much information as possible, including script output so
+    we can collect information about your configuration. This helps reduce the time required to
+    reproduce your issue.
+  * Check your issue regularly, as we may require additional information to successfully reproduce the
+    issue.
+* You may also open an issue to ask questions to the maintainers about whether a proposed change
+  meets the acceptance criteria, or to discuss an idea pertaining to the library.
+
+## Pull Request Guidelines ##
+
+By creating a pull request, you agree to the statements made in the [code license](#code-license) section. Your pull request 
```

Comment:
This link doesn't work for me; maybe it works after merge though. Or have to check .md links/.rst links if we switch to rst

Reply:
The link works for me, I stayed with .md, did not change to .rst.